lib/screens/headways.ex

```diff
@@ -124,16 +126,25 @@ defmodule Screens.Headways do
     ],
     red_ashmont: ~w[shmnl fldcr smmnl asmnl],
     red_braintree: ~w[nqncy wlsta qnctr qamnl brntn],
-    red_trunk: ~w[alfcl davis portr harsq cntsq knncl chmnl sstat brdwy andrw jfk]
+    red_trunk: ~w[alfcl davis portr harsq cntsq knncl chmnl sstat brdwy andrw jfk],
+    silver_chelsea: ~w[estav boxdt belsq chels]
   }
 
   # Mapping of parent station and route IDs to headway keys, for parent stations which serve more
   # than one line.
   @multi_stations %{
-    blue_trunk: {~w[Blue], ~w[state gover]},
+    blue_trunk: {~w[Blue], ~w[state gover aport]},
     green_trunk: {~w[Green-B Green-C Green-D Green-E], ~w[north haecl gover pktrm]},
     orange_trunk: {~w[Orange], ~w[north haecl state dwnxg]},
-    red_trunk: {~w[Red], ~w[pktrm dwnxg]}
+    red_trunk: {~w[Red], ~w[pktrm dwnxg]},
+    silver_seaport: {~w[741 742 746], ~w[conrd wtcst crtst sstat]},
+    silver_chelsea: {~w[743], ~w[conrd wtcst crtst sstat aport]}
+  }
+
+  @sl_multi_stations %{
+    # congress_st_at_wtc 17_096
+    silver_seaport: {~w[741 742 746], ~w[17096]},
+    silver_chelsea: {~w[743], ~w[17096]}
```

Comment:
It seems the only reason this needs to be a separate module attribute is that `17096` is a "standalone" stop and not a parent station, so it doesn't have the `place-` prefix which is otherwise assumed. To be clear about this we could call it `@multi_stops` (or something, it just shouldn't have "station" in the name IMO).